orchagent/intfsorch.cpp

```diff
@@ -365,8 +360,11 @@ void IntfsOrch::addIp2MeRoute(const IpPrefix &ip_prefix)
     attr.value.s32 = SAI_PACKET_ACTION_FORWARD;
     attrs.push_back(attr);
 
+    Port cpu_port;
+    gPortsOrch->getCpuPort(cpu_port);
```

Comment:
the function is void. if CPU port is not get successfully, portsOrch class is not initialized and orchagent will throw exception.

---
In reply to: [126777033](https://github.com/Azure/sonic-swss/pull/250#discussion_r126777033) [](ancestors = 126777033)